sql_generators/usage_reporting/templates/app_union.view.sql

```diff
@@ -0,0 +1,19 @@
+-- {{ header }}
+CREATE OR REPLACE VIEW
+  `{{ project_id }}.{{ app_name }}.{{ view_name }}`
+AS
+{% for channel in channels_info -%}
+{% if not loop.first -%}
+UNION ALL
+{% endif -%}
+SELECT
+  "{{ channel.channel_dataset }}" AS normalized_app_id,
+  -- {% if app_name == "fenix" and view_name == "usage_reporting_clients_daily" %}
+  -- mozfun.norm.fenix_app_info("{{ channel.channel_dataset }}", app_build).channel AS normalized_channel,
```

Comment:
@akkomar do we even need the UDF call here? My assumption here is that there were some bad builds in the past that required this, but considering this will be for new clients going forward is it enough if we just use the channel provided by probe scraper?

Reply:
Looking at the UDF source I agree: it shouldn't be needed. the app IDs are now distinct (and we only have 3 left)

Reply:
Thanks @akkomar and @badboy for the additional context. Then unless there are any objections I'd suggest we proceed without the UDF call and just use the value provided by probe-scraper. Considering this is only nightly right now I think this is okay and we can observe to see if we notice anything unexpected. Since this is quite a new thing we can easily tweak this later on.